attributes/default.rb

```diff
@@ -44,3 +47,9 @@
 default['flink']['parallelization']['degree']        = node['cpu']['total']
 default['flink']['webclient_port']                   = 8888
 default['flink']['taskmanager']['network_num_buffers']  = 2048
+
+#
+# Featurestore dependencies
+#
+default['flink']['hsfs']['version']                  = node['install']['version']
+default['flink']['hsfs']['url']                      = "#{node['download_url']}/hsfs/#{node['flink']['hsfs']['version']}/hsfs-flink-#{node['flink']['hsfs']['version']}.jar"
```

Comment:
Commented on Slack. I think we should avoid shipping the `hsfs-flink` jar in the distribution. I would prefer users upload it through their application.

I think it will create less changes of classpath collisions.